src/components/ICA/ICA.js

```diff
@@ -81,20 +107,36 @@ const ICA = ({
     numeral.locale(ICA.defaultProps.locale);
   }
 
-  const truncatedValue = truncateValue(percentage, value, truncate);
+  const truncatedValue = truncateValue(percentage, value, truncate, ICAClasses);
   const truncatedTotal = formatValue(total, truncate);
 
   const shouldDisplayof =
     (!percentage && total > value && truncatedValue !== truncatedTotal) ||
     (forceShowTotal && truncatedTotal > 0);
 
   return (
-    <div className={`${namespace} ${className}`} {...other}>
-      <h4 className={`${namespace}__label`}>{label} </h4>
-      <span className={`${namespace}__value`}>{truncatedValue}</span>
-      {shouldDisplayof ? (
-        <span className={`${namespace}__total`}> / {truncatedTotal}</span>
-      ) : null}
+    <div className={ICAClasses} {...other}>
+      <h4 className={`${namespace}__label ${ICAClasses}__label`}>{label} </h4>
+      <span className={`${namespace}__row`}>
+        {trending && (
+          <Icon
+            className={`${namespace}__trend ${ICAClasses}__trend`}
+            renderIcon={renderIcon}
+          />
+        )}
+        <span className={`${namespace}__value ${ICAClasses}__value`}>
+          {truncatedValue}
+        </span>
+        {shouldDisplayof ? (
+          <span
+            className={`${namespace}__total ${ICAClasses}__total ${trending &&
```

Comment:
Similarly to above, [`classnames`](https://www.npmjs.com/package/classnames#usage) can be leveraged to conditionally handle these classes.